src/unix/core.c

```diff
@@ -43,6 +43,13 @@
 #include <sys/utsname.h>
 #include <sys/time.h>
 
+#if defined(__has_feature)
```

Comment:
```suggestion
#if defined(__linux__) && defined(__has_feature)
```
TSAN is supported on more than just Linux.

Reply:
I used slightly complicated version that's reusable for other thread sanitizer detection. The clang has the header while gcc doesn't have the header (although it seems that linking works).

In the end, the wrappers are used only when clang is used on Linux.